### ISSUE TITLE
Failed to connect with certificates

### ISSUE THREAD

**Issue by tpimont** [nm-l2tp/network-manager-l2tp#88]
Hello,
First i make a connection to a windows 2016 server with PSK  -> OK
Then i extract certificates from Windows client machine and truncate in 2 pem files cert user and key user i add cacert too.
I just replace PSK with certificates TLS  -> KO

with certificate vpn dont connect .

**Comment by dkosovic** [comment]
NetworkManager-1.2.10 is the current stable version (and is in the _nm-1-2_ branch of this repository) has no certificate support at all.

The unstable master branch branch has some certificate support in the GUI, but no backend support as of yet. I started developing certificate support using NSS, but ran into issues due to limitations of what NSS supports. What I want to use is OpenSSL, but it is incompatible with the NetworkManager GPL license,  I was hoping OpenSSL with its new license would have been released by now.
